src/analysis/retail/evoker/augmentation/modules/talents/Accretion.tsx

```diff
@@ -22,10 +61,170 @@ class Accretion extends Analyzer {
+  }
+
+  private calcAccretionValue() {
+    const EbonMightUptime = this.selectedCombatant.getBuffUptime(
+      SPELLS.EBON_MIGHT_BUFF_PERSONAL.id,
+    );
+    const additionalUpheavalCastsViaCdr = Math.floor(this.effectiveUpheavelCDR / (36 * 1000));
+
+    const avgUpheavalCastDamage = this.totalUpheavelDamage / this.totalUpheavelCasts;
+
+    this.accretionUpheavel = avgUpheavalCastDamage * additionalUpheavalCastsViaCdr;
+
+    this.upheavelPercentage = (this.accretionUpheavel / this.totalDamageDone) * 100;
+
+    const avgShiftingSandsDamage =
+      this.totalShiftingSandsDamage / this.totalShiftingSandsApplications;
+
+    this.accretionShiftingSands = avgShiftingSandsDamage * additionalUpheavalCastsViaCdr;
+
+    this.shiftingSandsPercentage = (this.accretionShiftingSands / this.totalDamageDone) * 100;
+
+    const cdrUpheavelExtention =
+      (this.ebonMightUpheavelExtention / this.totalUpheavelCasts) * additionalUpheavalCastsViaCdr;
+
+    this.accretionEbonMight = (this.totalEbonMightDamage / EbonMightUptime) * cdrUpheavelExtention;
+
+    this.ebonMightPercentage = (this.accretionEbonMight / this.totalDamageDone) * 100;
+
+    if (debug) {
+      console.log('upheavel dam: ' + this.totalUpheavelDamage);
+      console.log('total damage: ' + this.totalDamageDone);
+      console.log('ebon might damage: ' + this.totalEbonMightDamage);
+      console.log('shifting sand damage: ' + this.totalShiftingSandsDamage);
+      console.log('effective cdr: ' + this.effectiveUpheavelCDR / 1000);
+      console.log('additional upheaval casts via CDR: ' + additionalUpheavalCastsViaCdr);
+      console.log('total upheavel casts: ' + this.totalUpheavelCasts);
+      console.log('avg upheaval cast damage: ' + avgUpheavalCastDamage);
+      console.log('cdr upheaval damage: ' + this.accretionUpheavel);
+      console.log('upheaval percentage: ' + this.upheavelPercentage);
+      console.log('avg shifting cast damage: ' + avgShiftingSandsDamage);
+      console.log('cdr shifting damage: ' + this.accretionShiftingSands);
+      console.log('shifting percentage: ' + this.shiftingSandsPercentage);
+      console.log('total em uptime: ' + EbonMightUptime);
+      console.log('em extension via upheaval: ' + this.ebonMightUpheavelExtention);
+      console.log('em extetion from extra casts: ' + cdrUpheavelExtention);
+      console.log('em damage via extension: ' + this.accretionEbonMight);
+      console.log('em percentage: ' + this.ebonMightPercentage);
+      console.log(
+        'accretion value: ' +
+          (
+            this.upheavelPercentage +
+            this.ebonMightPercentage +
+            this.shiftingSandsPercentage
+          ).toFixed(2) +
+          '%',
+      );
+    }
+  }
+
+  statistic() {
+    // If the talent didn't provide enough CDR to actually be relevant
+    // Then no reason to show it
+    if (this.accretionShiftingSands + this.accretionEbonMight + this.accretionUpheavel === 0) {
+      return null;
+    }
```

Comment:
I'm not sure this is how we should display it; isnt "dont play this talent on a fight this short" useful feedback? but also its fairly minor at the same time

Reply:
oh yea that's prolly better, will change that up

Reply:
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/3404958/832a75bd-41f1-452e-b3e6-db4c93536e17)